ql/src/java/org/apache/hadoop/hive/ql/txn/compactor/Worker.java

```diff
@@ -291,6 +302,13 @@ protected Boolean findNextCompactionAndExecute(boolean collectGenericStats, bool
         return false;
       }
 
+      if (ci.type.equals(CompactionType.REBALANCE) && t1.getSd().isSetNumBuckets() && t1.getSd().getNumBuckets() > 0) {
```

Comment:
I would prefer a helper method like, ci.IsRebalanceCompaction()

Reply:
what is `t1` do we have `t2` as well? Could we rename to `tbl` or `table`

Reply:
> I would prefer a helper method like, ci.IsRebalanceCompaction()

Answered above,

others addressed.